packages/admin-ui/src/table/components/styles/table-body.css.ts

```diff
@@ -0,0 +1,35 @@
+import { csx, dataAttr } from '@vtex/admin-ui-core'
+
+export const tableBodyTheme = csx({
+  display: 'contents',
+})
+
+export const tableBodyRowTheme = csx({
+  display: 'contents',
+  [dataAttr('clickable', 'true')]: {
+    cursor: 'pointer',
+    ':hover': {
+      '.__admin-ui-fixed-cell': {
```

Comment:
I wonder if it wouldn't be good to have something (a function or a constant) to handle these classnames.